qa/standalone/mon/osd-pool-df.sh

```diff
@@ -58,10 +58,11 @@ function TEST_ceph_df() {
     local ec_poolname=testcephdf_erasurecode
     create_pool $rep_poolname 6 6 replicated
     create_pool $ec_poolname 6 6 erasure ec42profile
+    sleep 2
```

Comment:
sleeps make me think the test may race - is there a way to wait for the desired state instead?